tests/integration/modules/test_cmdmod.py

```diff
@@ -262,7 +262,8 @@ def test_exec_code(self):
         '''
         cmd.exec_code
         '''
-        code = textwrap.dedent('''\
+        # `code` is a multiline YAML text. Formatting it as a YAML block scalar.
+        code = textwrap.dedent('''|
```

Comment:
Does `dedent` do anything here anymore? I'm pretty sure that with this leading `|` there's no leading spaces to dedent, and the output of this is

```
|
            import sys
            sys.stdout.write('cheese')
```

I'd either remove the misleading `dedent` call or rewrite it as

```
code = textwrap.dedent('''
    |
        import sys
        sys.stdout.write('cheese')
'''').lstrip()
```

(or, something else)

Reply:
@waynew @DmitryKuzmenko We should be using `tests.support.helpers.dedent` over `textwrap.dedent`.

Reply:
Wow. Thank you both!